public/lang/en.json

```diff
@@ -0,0 +1,564 @@
+{
+  "navigation": {
+    "module": "Modules",
+    "page": "Pages",
+    "chart": "Charts",
+    "automation": "Automation",
+    "configuration": "Configuration",
+    "adminPanel": "Admin panel",
+    "more": "More"
+  },
+
+  "block": {
```

Comment:
Restructure these keys to make sure we do not have any naming colisions with different block kinds.
Maybe like:

`block.style` => `block.general.style`
`block.title` => `block.general.title`
....

Also, `block.column` goes under `recordList` and `record`, just duplicate it...